scripts/alibaba_cloud_security_analyzer.py

```
@@ -0,0 +1,217 @@
+    print("Generating SARIF report...")
+    sarif_report = generate_sarif_report(access_findings, ecs_analysis)
+
+    # Output the report
+    output_file = "security-report.json"
+    with open(output_file, 'w') as f:
+        json.dump(sarif_report, f, indent=2)
+
+    print(f"Security analysis completed. Report saved to {output_file}")
+
+    # Print summary
+    print(f"\nSummary:")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)